Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "5d090e49dbd686d5b3ff43a06c7a54b2bbc6b1512c300022acf3df60ca934860"
+            "sha256": "7c51bf00baf06e2510a4c8a2afacd8c6899977d082c9afc93e7ce7ca7a2e7073"
```

Comment:
Not sure that this would be the right approach. We shouldn't add the project itself to Pipfiles and lock it.
If needed source can be installed in editable model by `pipenv install -e .`.
Although, I don't recall that it was necessary for coverage to work. Will look into it.

Reply:
Ah, ok, disregard my previous comment. I see now that the lock is only done on the lock file itself not on the project source code. Everything looks ok.